internal/usage/usage.go

```diff
@@ -107,6 +108,9 @@ const (
 	AccessListIps                   = "IP addresses to add to the new user’s access list."
 	StartDate                       = "Timestamp in ISO 8601 date and time format in UTC when the maintenance window starts."
 	EndDate                         = "Timestamp in ISO 8601 date and time format in UTC when the maintenance window ends."
+	AuthResult                      = "Flag that indicates whether to return either successful or failed authentication attempts. When set to true, Atlas filters the log to return only successful authentication attempts. When set to false, Atlas filters the log to return only failed authentication attempts."
```

Comment:
you say true or false here but the command says `success` and `fail` on the checks